spec/integration/api/posts_controller_spec.rb

```diff
@@ -0,0 +1,754 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Api::V1::PostsController do
+  let(:auth) {
+    FactoryGirl.create(
+      :auth_with_default_scopes,
+      scopes: %w[openid public:read public:modify private:read private:modify],
+      user: FactoryGirl.create(:user, profile: FactoryGirl.create(:profile_with_image_url))
```

Comment:
Layout/AlignHash: Align the elements of a hash literal if they span more than one line.